modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/pojo/index/ExperimentTagsByApplication.java

```diff
@@ -37,10 +38,14 @@
     @Column(name = "app_name")
     String appName;
 
+    @PartitionKey(1)
```

Comment:
@AndreaSuckro Pardon my lack of knowledge here! From the migration script looks like exp_id is just the clustering key and not the partition key. Do we declare clustering keys as a partition key in the datastax driver annotation like you've done here?